src/components/AnonymousChat/PostNotificationChannelItem.tsx

```diff
@@ -30,9 +30,6 @@ const PostNotificationChannelItem: (props: MessageChannelItemProps) => React.Rea
 
   const helperDetermineCommenterName = () => {
     const anonymousCommenterName = firstComment?.reaction?.data?.anon_user_info_emoji_name;
-    const firstCommenterId = firstComment?.reaction?.user_id;
-    const isFirstCommenterMe =
-      firstCommenterId === signedProfileId || firstCommenterId === anonProfileId;
 
     if (firstComment?.reaction?.isOwningReaction) {
       commenterName = 'You';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new code removes the logic for determining if the first commenter is the current user. This could potentially change the behavior of the application if there are other places in the code that rely on this information. Please ensure that this change does not affect other parts of the application.
```diff
-    const firstCommenterId = firstComment?.reaction?.user_id;
-    const isFirstCommenterMe =
-      firstCommenterId === signedProfileId || firstCommenterId === anonProfileId;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->